src/core/shaderlib/project64/project64.glsl.js

```diff
@@ -49,6 +49,14 @@ void project_position_fp64(vec4 position_fp64, out vec2 out_val[2]) {
   return;
 }
 
+void project_position_fp64(vec2 position, vec2 position64xyLow, out vec2 out_val[2]) {
```

Comment:
We should probably reaudit this module and see which functions/overloads should be public.